conf/defaults.ini

```diff
@@ -172,6 +172,9 @@ idle_conn_timeout_seconds = 90
 # If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request.
 send_user_header = false
 
+# Upper limit of bytes in the response body after http calls
+response_limit = 1000000
```

Comment:
Question is what the default should be. Either zero/disabled or something that makes sense for most of the users. Risk is that putting it too low introduces a breaking change. Would vote for zero now. @bergquist @dsotirakis  any thoughts?

```suggestion
# Limit the amount of bytes that will be read/accepted from responses of outgoing HTTP requests.
response_limit = 0
```

Reply:
`0` for me as well - was thinking about an arbitrary big number so we won't fail, but since we've introduced the ability to move to the next request through `RoundTripper(next)` is fine to have `0` as a default limit 👍